packages/design-tokens/README.md

```diff
@@ -2,142 +2,94 @@
+4. [Rebranding Spirit](#rebranding-spirit)
 5. [FAQ](#faq)
+6. [License](#license)
 
 ## Available Design Tokens
 
-| Category      | Supernova | Figma | Sass                 |
-| ------------- | --------- | ----- | -------------------- |
-| 🖼 Borders    | ✅        | ❌    | [`_borders.sass`]    |
-| 🎨 Colors     | ✅        | ✅    | [`_colors.sass`]     |
-| 🖌️ Gradients  | ✅        | ✅    | [`_gradients.sass`]  |
-| 📏️ Measures  | ✅        | ❌    | [`_measures.sass`]   |
-| ⚙️ Other      | ✅        | ❌    | [`_other.sass`]      |
-| 🎱 Radii      | ✅        | ❌    | [`_radii.sass`]      |
-| ⛱ Shadows    | ✅        | ✅    | [`_shadows.sass`]    |
-| 🔠 Typography | ✅        | ✅    | [`_typography.sass`] |
+Tokens are split into two categories: Global and Themes. Global tokens are
+shared across all themes, while theme tokens are specific to a particular theme.
 
-## Install
+They are managed in Figma and exported to Supernova.
 
-🙋🏻‍♂️ **Hold on! Do you already use [`spirit-web`]?** Then you don't need to
-install this package because `spirit-design-tokens` is installed automatically
-as a dependency of [`spirit-web`].
+All content in `src` is generated by Supernova and should not be edited manually.
 
-If you want to use just `spirit-design-tokens` alone in your project, run:
+In `scss` directory are files `@global.scss` and `@themes.scss` which contain links to all available tokens.
 
-```shell
-yarn add @lmc-eu/spirit-design-tokens
-```
+### Global Tokens
 
-or
+The category consist of these groups:
 
-```shell
-npm install --save @lmc-eu/spirit-design-tokens
-```
+- Borders
+- Gradients
+- Other
+- Radii
+- Shadows
+- Typography
 
-## Basic Usage
+These tokens does not depend on any theme and are shared across all themes.
 
-The recommended approach in Sass is to import all Spirit design tokens at once
-via the [`@tokens` API](#tokens-api):
+### Theme Tokens
 
-```scss
-@use 'sass:map';
-@use 'node_modules/@lmc-eu/spirit-design-tokens/scss/@tokens' as tokens;
-
-.MyComponentThatMightGoToSpiritInFuture {
-  font-family: map.get(tokens.$body-medium-text-regular, font-family);
-  color: tokens.$text-primary-default;
-}
-```
+You can find the list of themes in the `src/themes` directory. Each theme has
+its own directory and set of design tokens.
 
-This makes it easier to [migrate your code to Spirit][migrate-to-spirit] in the
-future.
+The category currently consist only of this group:
 
-<details>
-<summary>Optional: import by categories</summary>
+- Colors
 
-You can also import individual design token files by categories, e.g.:
+Each theme consists of the same set of tokens, but with different values.
 
-```scss
-@use 'sass:map';
-@use 'node_modules/@lmc-eu/spirit-design-tokens/scss/colors';
-@use 'node_modules/@lmc-eu/spirit-design-tokens/scss/typography';
-
-.MyComponent {
-  font-family: map.get(typography.$body-medium-text-regular, font-family);
-  color: colors.$text-primary-default;
-}
-```
+We currently support two themes:
```

Comment:
This is the first think we forgot to update. Can we just point to the file with available themes?

Reply:
I think it is helpful to show what themes we have. The only change we will make is to introduce dark mode and we will have to document it here as well, so we can modify it.

I am pretty sure we won't add more themes because our Figma plan allows only 4 of them :D

Reply:
Ok.